sitecrafting/gearlab-tools-wordpress
# Support for Timber templates

**TO TEST:**

Clone and checkout this branch:

```
git clone git@github.com:sitecrafting/gearlab-tools-wordpress.git
cd gearlab-tools-wordpress
git checkout feature/templates
```

Build a release archive from source:

```
bin/build-release.sh --no-interaction vtest-templates
```

This creates a zip (and tar) file like the ones available for download on the releases page. Unzip to a project directory like a regular plugin.

Instructions for configuring credentials and such are in the README (and should be up to date on this branch). **Once installed, activated, and configured on your project, the following should work:**

* Performing a search on the frontend with no template overrides (no new theme files)
* Performing a search when overriding any of the following (paths relative to theme root):
  * `/gearlab-tools/search.php`
  * `/views/gearlab-tools/search.twig`
  * `/views/gearlab-tools/search-result.twig`
